src/pyOpenMS/pxds/EmgGradientDescent.pxd

```diff
@@ -12,5 +13,5 @@ cdef extern from "<OpenMS/MATH/MISC/EmgGradientDescent.h>" namespace "OpenMS":
 
         void getDefaultParameters(Param&) nogil except +
 
-        void fitEMGPeakModel(const MSChromatogram& input_peak, MSChromatogram& output_peak) nogil except +
-        void fitEMGPeakModel(const MSSpectrum& input_peak, MSSpectrum& output_peak) nogil except +
+        void fitEMGPeakModel(MSChromatogram input_peak, MSChromatogram& output_peak) nogil except +
```

Comment:
@hroest How come the removal of "&" from the input? And what about const? I thought they were both legal

Reply:
you are right in this case it does not make a big difference. I general adding `&` can incur large overheads on the pyOpenMS wrappers because they copy data back even though the object did not change. Here both would work

Reply:
What do you (or Cython team?) recommend doing when passing input data that doesn't change? Should I use `const` and a reference (`&`) or not?
Cythong docs don't really help me...

Reply:
yes, but autowrap does not yet fully support `const ref` at the moment.

Reply:
I am working on proper support for `const X &`, see https://github.com/uweschmitt/autowrap/pull/70